go/signedexchange/README.md

```diff
@@ -1,3 +1,83 @@
-# gen-signedexchange
+# go/bundle
+This directory contains a reference implementation of [Signed HTTP Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html) spec.
 
-This directory will hold a reference implementation of a tool to sign exchanges, but for now, the version that best matches Chrome is in https://github.com/nyaxt/webpackage/tree/master/go/signedexchange .
+## Overview
+We currently provide two command-line tools: `gen-signedexchange` and `gen-certurl`.
+
+`gen-signedexchange` generates a signed exchange file. The `gen-signedexchange` command constructs HTTP request and response pair from given command line flags, attach a cryptographic signature of the pair, and serializes the result to output file.
```

Comment:
nit: constructs HTTP request ... -> constructs an HTTP request and response pair

attach a -> attaches the

to output file -> to an output file, or to the given output file

Reply:
Done